herd/machAction.ml

```diff
@@ -322,6 +322,11 @@ end = struct
      end
   | _ -> false
 
+  let has_cmodx_restrictions = function
```

Comment:
Feel free to ignore this, but Isn't it possible to move this to `herd/AArch64Arch_herd.ml`? CMODX is Arm specific and it would be nice if we didn't have in machAction.ml. You can define a new like barrier_sets or cmo_sets

Reply:
Thanks -- I implemented this in commit 2866f46. What do you think?